README.md

```diff
@@ -66,12 +67,12 @@ http://web.eecs.umich.edu/~fessler/irt/reproduce/
 
 ## Compatibility
 
-Current version is compatible with Julia 1.3.
```

Comment:
FWIW, MIRT is actually not compatible with Julia 1.3 because one of its dependency, i.e., LinearMapAA, isn't. Do you want to make it compatible with Julia 1.3, or just remove this from here?

```suggestion
Current version is compatible with Julia 1.4.
```

Reply:
Oh right.  Fixed...